modules/v4l2/v4l2.c

```diff
@@ -476,7 +495,7 @@ static int alloc(struct vidsrc_st **stp, const struct vidsrc *vs,
 		return EINVAL;
 
 	if (!str_isset(dev))
-		dev = "/dev/video0";
+		dev = ((struct mediadev*)list_head(&vs->dev_list)->data)->name;
```

Comment:
perhaps add a helper function in mediadev.c that will return the first device
or the name of the first device ?